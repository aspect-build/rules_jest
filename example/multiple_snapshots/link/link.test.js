import renderer from "react-test-renderer";
import Link from "./link";

it("renders correctly", () => {
  const tree = renderer
    .create(<Link page="https://aspect.build">Aspect</Link>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
