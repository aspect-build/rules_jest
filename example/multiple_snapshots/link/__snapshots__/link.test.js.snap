// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  href="https://aspect.build"
>
  Aspect
</a>
`;
